content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -32,22 +32,59 @@ LoadBalancer, or with dynamic PersistentVolumes.
 
 ## {{% heading "prerequisites" %}}
 
+The prerequisites depend on which topology you have selected for your cluster's
+control plane:
 
-For both methods you need this infrastructure:
+{{< tabs name="prerequisite_tabs" >}}
+{{% tab name="Stacked etcd" %}}
```

Comment:
> duplicating the content in tabs opens up for de-dup mistakes

It does, but on the other hand I'd like to shift the effort in understanding to us (once) rather than readers (many times).
I found the original / current page not that straightforward to follow.

If the main concern is about duplicated changes, I can add an HTML comment to highlight that concern to future reviewers.

Reply:
(and have done)

Reply:
> It does, but on the other hand I'd like to shift the effort in understanding to us (once) rather than readers (many times).

not sure what you mean by "once" vs "many times".

> I found the original / current page not that straightforward to follow.

could you elaborate why?
it think the steps are rather simple, "1. for both methods you need this, 2. only for method Foo you need this additional step"

i would like to get a second opinion from the Docs maintainers on this one.
do you have someone in mind that we can ask?

> If the main concern is about duplicated changes, I can add an HTML comment to highlight that concern to future reviewers.

yes, it's a minor concern. both reviewers and authors may entirely miss the comment an introduce drift in the two tabs. which can then result in further noise in this repository. new issues that highlight the drift, PRs to fix it, etc.

instead of a comment, does hugo have a way to copy a predefined block between multiple areas of a document?

Reply:
> instead of a comment, does hugo have a way to copy a predefined block between multiple areas of a document?

Yes (we can set up some headless bundles and make a shortcode), but it feels like more complexity than is justified.

Reply:
> i would like to get a second opinion from the Docs maintainers on this one.

We have a policy that for changes that go live when merged, approvers _don't_ approve their own PR. So we always have an approver review changes to the live docs even if the author is an approver.

This change isn't urgent; I'm happy to let this sit in with [other PRs](https://github.com/kubernetes/website/pulls/sftim) I have open.

Reply:
I might split that commit out into its own PR.